webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -1232,7 +1236,7 @@ var userOverride = {
             MESSAGE_BUBBLE_AVATOR_ENABLED =
                 typeof optns.messageBubbleAvator === 'boolean'
                     ? optns.messageBubbleAvator
-                    : false;
+                    : true;
```

Comment:
where is this used?

Reply:
if  `MESSAGE_BUBBLE_AVATOR_ENABLED = true` then `mck-msg-avator-bubble` get appended to message container div 

if `mck-msg-avator-bubble` is present in the message container div then only the timestamp for a message appears below the message otherwise it appears beside the message. (as shown below)
![Screenshot 2021-10-04 at 4 27 10 PM](https://user-images.githubusercontent.com/19753380/135839760-deab7cbc-a5eb-4dd3-8d73-7003550bb3c2.png)

Reply:
the default value for MESSAGE_BUBBLE_AVATOR_ENABLED in normal flow is true, but for reset flow it was false. 
was this done intentionally?

Reply:
don't remember any, might be a mistake